rholang/main/runner.py

```diff
@@ -4,9 +4,10 @@
 from contextlib import contextmanager
 from io import StringIO
 from os.path import relpath
+from threading import Timer
```

Comment:
Ugh. Threads. I try to stick to event loops just like I try to maintain ocap discipline.

So I wish we could use [twisted schedule events](https://twistedmatrix.com/documents/current/core/howto/time.html).

But I guess we're doing django, not twisted. oh well.